step2/try2.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import "fmt"
+
+func main() {
+	//条件分岐にifを使用する場合
+	for i := 1; i <= 4; i = i+1{
```

Comment:
`i = i+1`っていうのはあまり見ないので、 `i++`にした方が可読性が上がるね！
あと、コードフォーマットが聞いていないので、またそちらについてはGolandの使い方として教えるね！

```suggestion
	for i := 1; i <= 4; i++ {
```

Reply:
そうなんですね！（確かに、テキストの回答でもi++とされていました…）
修正しました！